ConwaysGameOfLife.Tests/LocationShould.cs

```diff
@@ -8,7 +8,7 @@ public class LocationShould // TODO: all tests relate to universehelper methods
     {
         [Theory]
         [InlineData(0, 0, 0, 0)]
-        public void WhenTwoLocationsInstantiated_BeEqual(int x1, int y1, int x2, int y2)
+        public void GivenTwoIdenticalLocations_BeEqual(int x1, int y1, int x2, int y2) // TODO: name very close to test on line 34 9/6
         {
             var location1 = new Location(x1, y1);
             var location2 = new Location(x2, y2);

```

Comment:
- Naming suggestion
- Can simplify params to take just one x and y value
- Add another test case or two

```suggestion
        [InlineData(5,1)]
        public void BeEqual_GivenTwoLocationsOfSameCoordinates(int x, int y) // TODO: name very close to test on line 34 9/6
        {
            var location1 = new Location(x, y);
            var location2 = new Location(x, y);
```